src/views/AnnualReport.vue

```diff
@@ -30,7 +30,7 @@
     />
 
     <bcol-error-dialog
-      :bcolOBject="bcolObj"
+      :bcolObject="bcolObj"
```

Comment:
woah, how did this work before with the case being wrong :s Good catch!

Reply:
Apparently, props aren't case sensitive. See https://www.google.com/search?q=vue+prop+name+casing .

There's also some magic with event names ... something like ImAnEvent gets transformed to im-an-event ... not sure. I just follow our existing code style, which works :)